packages/jest-cli/src/runGlobalHook.js

```diff
@@ -6,9 +6,22 @@
  *
  * @flow
  */
+
 import type {GlobalConfig} from 'types/Config';
 import type {Test} from 'types/TestRunner';
 
+import {extname, resolve} from 'path';
+import pEachSeries from 'p-each-series';
```

Comment:
using `p-each-series` allows us to make sure one require hook does not interfere with another